src/Files.App/Utils/Git/GitHelpers.cs

```diff
@@ -78,7 +76,7 @@ private set
 
 			try
 			{
-				if (Repository.IsValid(path))
+				if (SafetyExtensions.IgnoreExceptions(() => Repository.IsValid(path)))
```

Comment:
It could be left as is, but it would be even better to cut it out into a separate method and call it.

Reply:
What's the benefit of doing it like that?

Reply:
Don't have to surround each call to `isValid` with `SafetyExtensions.IgnoreExceptions`.

Reply:
Done 👍